db/migrate/20130920225910_create_teammates.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTeammates < ActiveRecord::Migration
+  def change
+    create_table :teammates do |t|
+      t.references :event, index: true
+      t.references :user, index: true
+      t.boolean :notifications, default: true
+      t.string :role
+      t.string :email
+      t.string :state
+      t.string :slug
```

Comment:
since we dropped slug from the invite path, do we have a reason to have it on the teammates table?  If not, let's remove from everywhere.